mu-plugins/blocks/ratings-stars/render.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+$attributes = apply_filters( 'wporg_block_ratings_stars', $attributes );
```

Comment:
Pretty much all the same comments apply to this block too— naming scheme, whether this should filter all of `$attributes` or just a placeholder value (like query total & query filters do), and it should also get the post ID from context to pass back.